common/app/views/fragments/containers/facia_cards/standardHeaderMeta.scala.html

```diff
@@ -19,7 +19,7 @@
                     "tone-comment" -> isContributor,
                     "tone-colour" -> isContributor))">
                     @href.map { href =>
-                    <a data-link-name="section heading" href="@LinkTo {/@href}">
+                    <a data-link-name="section heading" href="/@LinkTo(href, None)">
```

Comment:
This is actually the problem. It is used to display all container, which mean all title in front would be redirected with this change. 
I am gonna sleep on it and see if I can come up with a good solution